src/events/mouse.js

```diff
@@ -466,6 +466,8 @@ p5.prototype._setMouseButton = function(e) {
  * Browsers may have different default
  * behaviors attached to various mouse events. To prevent any default
  * behavior for this event, add "return false" to the end of the method.
+ * Otherwise add "return true" if you wish for default
+ * mouse behavior to be recognized by browser.
```

Comment:
I thought that the default behavior would occur if there was no return line. Or do I have that wrong?

Reply:
Returning nothing should give the default behavior as the code only check for false with `===` so any other value (including `undefined` when there is no return) should give default behaviour